pkg/storage/storage.go

```diff
@@ -10,6 +10,18 @@ import (
 	"github.com/pingcap/kvproto/pkg/backup"
 )
 
+// WalkOption is the option of storage.WalkDir.
+type WalkOption struct {
+	// walk on SubDir of specify directory
+	SubDir string
+	// number of list count, default 1000
+	ListCount int64
+	// this is used for s3 storage, when walk on specify directory,
+	// the result include storage.Prefix, which can not be reuse in other API.(Open/Read),
+	// we can set RemovePrefix to true, to avoid this problem.
+	RemovePrefix bool
```

Comment:
when do we want this option to be `false`?

Reply:
Maybe in localStroage 🤔️，it depends on our implements of Read/Open API. Since out S3 Read API use prefix internally already.

Reply:
Local storage also uses relative path since #459

Reply:
So,  we can remote it, and then always TrimPrefix for each path?

Reply:
Yes

Reply:
OK, Done